sections/ui_fullTable.R

```diff
@@ -17,8 +17,9 @@ body_fullTable <- dashboardBody(
   ),
   fluidPage(
     fluidRow(
-      h3(HTML(paste0("&Uuml;bersichtstabelle (", strftime(current_date, format = "%d.%m.%Y"), ")")),
-        class = "box-title", style = "margin-top: 10px; font-size: 18px;"),
+      #h3(HTML(paste0("&Uuml;bersichtstabelle (", strftime(current_date, format = "%d.%m.%Y"), ")")),
```

Comment:
This would be more in line with the overview section
```suggestion
      h3(HTML(paste0("&Uuml;bersichtstabelle - ", strftime(input$overviewSlider, format = "%d.%m.%Y"))),
```

Reply:
the input$overviewSlider option gives me an error saying 'input' not found

Reply:
Ah I see. That makes sense, because it is source outside of a reactive context. I see. Then maybe we can just adjust the formatting a little so it is according to the overview page